exporter/loggingexporter/logging_exporter_test.go

```diff
@@ -28,28 +28,24 @@ import (
 
 func TestLoggingTraceExporterNoErrors(t *testing.T) {
 	lte, err := NewTraceExporter(&configmodels.ExporterSettings{}, zap.NewNop())
-	if err != nil {
-		t.Fatalf("Wanted nil got %v", err)
-	}
+	assert.NotNil(t, lte)
```

Comment:
This needs to use `require` instead of `assert` otherwise the test will crash if nil is returned. Same in other similar places.